### DISCUSSION TITLE
LXQt on Mastodon

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2325] [Media]
Time ago I created also a [Mastodon account](@LXQt@mastodon.social), which didn't get much attention neither from me nor from people. With the recent events happening at Twitter I made a tweet about it saying that it will be in sync with twitter and I think we should add the link to the website too.

As I know very little about Mastodon I created it as @LXQt@mastodon.social,  maybe it should be migrated on https://fosstodon.org/about, no  clear idea about how important the server used really is.

**Comment by stefonarch** [comment]
Decided to let it be at it is, a link is added on the website.
A nice feature is that one can follow a tag in a feedreader:
![screen_area_sab_21:13:12_](https://user-images.githubusercontent.com/10681413/204107393-fadc9e18-ae84-4d51-890b-0c58b084450e.png)

https://mastodon.social/tags/lxqt.rss

**Comment by edwardp3** [comment]
Congrats on joining Mastodon. I joined it a few weeks ago and am still trying to work my way around everything, how this or that works, etc.
